docs/user_guide.md

```diff
@@ -121,13 +139,18 @@ Each light blue bar represents the contribution timeline of an individual author
 - As Figure shown, when the user hovers the mouse above a ramp, the time period and the exact amount of contribution will be shown.
```

Comment:
This part sounds wrong. Should be 'In the figure shown' or 'As the figure shows'. 
Also, user hovers the mouse ~~above~~ over a ramp